website/src/types/Config.ts

```diff
@@ -1,5 +1,7 @@
 export interface BrowserConfig {
   ENABLE_CHAT: boolean;
+  ENABLE_DRAFTS_FOR_PLUGINS: boolean;
```

Comment:
Since the meaning of this configuration option is not immediately clear (at least it was not fro me quickly reading through the PR) it would be good to have a comment here which explains what it does.